packages/collector/test/tracing/cloud/aws-sdk/v3/sqs/receiver.js

```diff
@@ -38,10 +38,16 @@ const agentPort = process.env.INSTANA_AGENT_PORT || 42699;
 const sqsV3ReceiveMethod = process.env.SQSV3_RECEIVE_METHOD || 'v3';
 const queueURL = process.env.AWS_SQS_QUEUE_URL;
 const awsRegion = 'us-east-2';
-
-const sqs = new awsSdk3.SQSClient({ region: awsRegion });
-const sqsv2 = new awsSdk3.SQS({ region: awsRegion });
-
+let sqs;
+let sqsv2;
+
+if (process.env.AWS_ENDPOINT) {
+  sqs = new awsSdk3.SQSClient({ region: awsRegion, endpoint: process.env.AWS_ENDPOINT });
+  sqsv2 = new awsSdk3.SQS({ region: awsRegion, endpoint: process.env.AWS_ENDPOINT });
```

Comment:
FYI: This is really nice, because we proofed that localstack is working with SNS & SQS.